packages/fungible-token/bridge-fungible-token/src/data_structures.sw

```diff
@@ -38,9 +38,12 @@ impl MessageData {
         };
 
         // TODO: Bug, have to mutate this struct for these values or tests fail
-        msg_data.amount = input_message_data(msg_idx, OFFSET_AMOUNT).into();
-        msg_data.from = input_message_data(msg_idx, OFFSET_FROM).into();
-        msg_data.token_id = input_message_data(msg_idx, OFFSET_TOKEN_ID).into();
+        msg_data.amount = input_message_data(msg_idx, OFFSET_AMOUNT)
+            .into();
```

Comment:
I would consider this as a fmt bug. Perhaps raising an issue for discussion is good

Reply:
Yeah this looks like a formatting bug. @Braqzen would you mind opening an issue for this?

Reply:
Sure, I'll create a minimal repro today